### PR TITLE
pipewire: 0.3.34 -> 0.3.35

### PR DESCRIPTION
**Pull request by jansol**
###### Motivation for this change

https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.35

Other than S/PDIF passthrough, mainly compatibility and quality of life improvements.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Kranzes — APPROVED** [review]

**Comment by jtojnar** [comment]
> Bluetooth codecs are now compiled into sperate plugins which are dynamically loaded. This makes it possible to change the plugin implementation or ship plugins separately without having to recompile the bluetooth module.

Can someone verify that it uses a correct directory (since the package has multiple outputs).

**Comment by jtojnar** [comment]
Also, what about the manpage generation?

**Comment by jansol** [comment]
> > Bluetooth codecs are now compiled into sperate plugins which are dynamically loaded. This makes it possible to change the plugin implementation or ship plugins separately without having to recompile the bluetooth module.
> 
> Can someone verify that it uses a correct directory (since the package has multiple outputs).

They are in the `pipewire.lib` output, under `lib/spa-0.2/bluez5` just like the main bluez5 plugin.

> Also, what about the manpage generation?

I don't think that was ever discussed? Which output do they even belong to?

FWIW the HTML docs *are* currently generated into the doc output.

**Comment by jtojnar** [comment]
They usually go either to `out` or `man` output. I am mentioning them because they now require `rst2man` so they are not built anymore (not sure if they were before).

**Comment by jansol** [comment]
Ah right, they weren't built in the past because of the xmltoman dependency. Looks like everything is peachy now.

File: pkgs/development/libraries/pipewire/default.nix [hunk 1]
```diff
@@ -125,7 +128,7 @@ let
 
     mesonFlags = [
       "-Ddocs=enabled"
-      "-Dman=disabled" # we don't have xmltoman
+      "-Dman=enabled"
```
Comment (jtojnar) [inline pkgs/development/libraries/pipewire/default.nix:131]: This line can be dropped, we enable autofeatures by default.

**Review by Kranzes — APPROVED** [review]

File: pkgs/development/libraries/pipewire/default.nix (hunk 1 above)
Comment (jansol) [reply pkgs/development/libraries/pipewire/default.nix]: (done)

**Comment by jansol** [comment]
@jtojnar mind doing the honors? Or is there something else that still needs changing.

**Pull request merged**